lib/pure/collections/sequtils.nim

```diff
@@ -80,29 +80,6 @@ import macros
 when not defined(nimhygiene):
   {.pragma: dirty.}
 
-
-macro evalOnceAs(expAlias, exp: untyped,
```

Comment:
instead of deleting it, hide this as dead code behind a `when false:`, it's tricky to implement and may be useful in other contexts later.
(when false is better than a comment also, to keep syntax highlighting, checking for syntactic validity and less comment noise)

Reply:
If you want to keep it alive, put it in a repository with utils you care about. I personally am very happy to get rid of it.

Reply:
causes regressions https://github.com/nim-lang/Nim/pull/13687#issuecomment-602307527